spec/spec.md

```diff
@@ -885,6 +885,12 @@ Descriptor Objects_ are composed as follows:
         Setting the property to `false`, or omitting the property, indicates
         the processing entity ****MAY**** submit a response that contains more
         than the data described in the `fields` array.
+      - The object ****MAY**** contain a `self_attested` property, and if
+        present its value ****MUST**** be one of the following strings:
+        - `required` - This indicates that the processing entity ****MUST****
+          submit a response provided by the Holder.
```

Comment:
hmm, I'm not sure this language is clear enough... In my mind `self_attested`, means that the issuer of the credential and the holder of the credential are the same... the subject could be anything... I can attest to daniel working for microsoft, and present that credential... in that case.

1. daniel is the subject
2. the claim is microsoft employee
3. the issuer is orie
4. the holder is orie.

compare that to...

1. daniel is the subject
2. the claim is microsoft employee
3. the issuer is pam
4. the holder is orie.

the language needs to be clearer.